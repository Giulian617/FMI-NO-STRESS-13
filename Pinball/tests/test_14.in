100 38770378449891
63 21 1
48 63 9
75 48 6
79 75 10
25 79 7
42 25 10
12 42 3
2 12 9
51 2 2
62 51 5
41 62 10
70 41 3
8 70 10
13 8 3
38 13 5
37 38 9
54 37 10
58 54 2
99 58 9
57 99 9
4 57 2
84 4 3
68 84 1
45 68 9
32 45 2
47 32 2
60 47 4
78 60 1
100 78 1
6 100 4
61 6 3
93 61 1
24 93 3
27 24 7
85 27 3
91 85 2
56 91 2
98 56 9
83 98 6
95 83 2
66 95 1
80 66 2
96 80 10
30 96 1
89 30 1
71 89 2
35 71 4
34 35 1
17 34 9
64 17 6
52 64 7
9 52 4
82 9 6
5 82 6
26 5 2
90 26 7
76 90 5
15 76 3
14 15 2
81 14 4
77 81 6
31 77 4
74 31 2
11 74 1
39 11 8
87 39 3
55 87 8
88 55 5
18 88 4
65 18 3
10 65 2
43 10 6
73 43 6
7 73 3
22 7 5
28 22 2
33 28 6
40 33 6
16 40 8
1 16 1
86 1 8
36 86 1
69 36 4
72 69 5
67 72 8
3 67 8
29 3 2
46 29 8
49 46 3
92 49 6
97 92 7
94 97 7
53 94 1
50 53 8
23 50 7
59 23 10
44 59 6
20 44 9
19 20 2
