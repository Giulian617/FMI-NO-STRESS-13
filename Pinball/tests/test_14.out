111111112
888888896
